packages/mason/test/src/generator_test.dart

```diff
@@ -12,6 +12,10 @@ class _MockLogger extends Mock implements Logger {}
 
 void main() {
   group('MasonGenerator', () {
+    final identicalString = '  ${cyan.wrap('identical')} ';
```

Comment:
These tests were acting a bit flaky so I made them a bit more strict which seemed to help